tests/collectioncardmakertest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+require('../src/CollectionCardMaker.php');
+
+use PHPUnit\Framework\TestCase;
+
+class CollectionCardMakerTest extends TestCase {
+
+
+public function testcreateGameCard()
+{
+   
+    $name = 'wow';
+    $playercount = '2 - 4';
+    $difficulty = '2';
+    $description = 'blah';
+
+    $creategamecard = new CollectionCardMaker($name, $playercount, $difficulty, $description);
```

Comment:
if this class is renamed as per my previous comment, then this variable will need a more semantic name (feel free to ask for hints)